backend/models/db/answer-attachments.model.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
We can probably do away with this file since we now store attachments on aws and a reference to those attachments on the surveyService